.gitignore

```diff
@@ -2,3 +2,4 @@
 /.build
 /Packages
 /*.xcodeproj
+/.idea
```

Comment:
This is for IDEs right?

Reply:
(By which I mean to ask: is it needed? It's been a long while since I've used IDEs, so I don't know how they interact with package managers)